runtime-modules/content/src/benchmarks/benchmarking.rs

```diff
@@ -1797,6 +1792,642 @@ benchmarks! {
+        let video_id = setup_video_with_nft_in_buy_now::<T>(curator_account_id, actor, channel_id, old_price)?;
+        let new_price = BalanceOf::<T>::from(2u32);
+    }: _ (origin, actor, video_id, new_price)
+        verify {
+            assert!(matches!(Pallet::<T>::video_by_id(video_id).nft_status, Some(Nft::<T> {
+                transactional_status: TransactionalStatus::<T>::BuyNow(new_price),
+                ..
+            })));
+        }
+
+    // WORST CASE SCENARIO:
+    // STATE COMPLEXITY
+    // - curator owned channel
+    // - curator number is max
+    // - curator has max number of permissions
+    // - channel has all features paused except necessary ones
+    // - channel has max assets
+    // - NFT owner == channel owner
+    // - NFT royalty is some
+    // INPUT COMPLEXITY
+    buy_nft {
+        let (channel_id, group_id, lead_account_id, curator_id, curator_account_id) =
+            setup_worst_case_scenario_curator_channel_all_max::<T>(false)?;
+        let actor = ContentActor::Curator(group_id, curator_id);
+        let price = BalanceOf::<T>::from(100u32);
+        let video_id = setup_video_with_nft_in_buy_now::<T>(curator_account_id, actor, channel_id, price)?;
+
+        let (buyer_account_id, buyer_id) = member_funded_account::<T>(DEFAULT_MEMBER_ID);
+        let origin = RawOrigin::Signed(buyer_account_id.clone());
+
+        let balance_pre = Balances::<T>::usable_balance(buyer_account_id.clone());
+    }: _ (origin, video_id, buyer_id, price)
+        verify {
+            assert!(matches!(Pallet::<T>::video_by_id(video_id).nft_status, Some(Nft::<T> {
+                transactional_status: TransactionalStatus::<T>::Idle,
+                ..
+            })));
+            assert_eq!(Balances::<T>::usable_balance(buyer_account_id), balance_pre - price)
+        }
+
+    // ================================================================================
+    // ========================== NFT - UPDATE LIMITS =================================1
+    // ================================================================================
+
+    // WORST CASE SCENARIO:
+    // INPUT COMPLEXITY
+    toggle_nft_limits {
+        let origin = RawOrigin::Root;
+        let enabled = false;
+    }: _ (origin, enabled)
+        verify {
+            assert!(!Pallet::<T>::nft_limits_enabled());
+        }
+
+    // STATE COMPLEXITY
+    // INPUT COMPLEXITY
+    update_global_nft_limit {
+        let origin = RawOrigin::Root;
+        let nft_limit_period = NftLimitPeriod::Daily;
+        let limit = 10u64;
+    }: _(origin, nft_limit_period, limit)
+        verify {
+            assert_eq!(Pallet::<T>::global_daily_nft_limit().limit, 10u64);
+        }
+
+    // STATE COMPLEXITY
+    // - curator owned channel
+    // - curator number is max
+    // - curator has max number of permissions
+    // - channel has max collaborators
+    // - channel has all features paused except necessary ones
```

Comment:
```suggestion
    // - channel has all features paused
```
There are no necessary ones in this case.
Actually the only extrinsic in the nft group that requires some feature to **not** be paused is `issue_nft`